### DISCUSSION TITLE
Help Identifying Forks with my Proprietary Information in them

### DISCUSSION THREAD

**Discussion by madamsxyz** [community/community#111150] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm in the process of submitting a request to take down proprietary information from a public repository but I do not know if there are also forks that could include this proprietary info (but I have reason to believe there could be).  Is there any GitHub support available to assist with identifying forks that include this proprietary info from the public repository we discovered?

**Comment by AjayBadrinath** [comment]
Hi there Hope your issue is resolved ! You can check the public repo forks on github.com/<user>/<repo_name>/forks and you can submit a DMCA request for the same https://docs.github.com/en/site-policy/content-removal-policies/guide-to-submitting-a-dmca-takedown-notice
